app/src/main/java/com/duycuong/weather/ui/screen/main/current_weather/CurrentWeatherViewModel.java

```diff
@@ -0,0 +1,109 @@
+package com.duycuong.weather.ui.screen.main.current_weather;
+
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.location.Location;
+import android.widget.Toast;
+
+import com.duycuong.weather.BR;
+import com.duycuong.weather.R;
+import com.duycuong.weather.data.model.CurrentWeather;
+import com.duycuong.weather.data.model.CurrentlyResponse;
+import com.duycuong.weather.data.source.WeatherDataSource;
+import com.duycuong.weather.data.source.WeatherRepository;
+import com.duycuong.weather.data.source.remote.WeatherRemoteDataSource;
+import com.duycuong.weather.utils.Constant;

```

Comment:
ERROR: Unused import: com.duycuong.weather.utils.Constant. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports